src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -26,12 +27,10 @@ class FormStartControls extends React.Component {
 
   captureAnalytics = () =>
     this.props.gaStartEventName &&
-    window.dataLayer.push({ event: this.props.gaStartEventName });
+    recordEvent({ event: this.props.gaStartEventName });
 
   handleLoadPrefill = () => {
     this.captureAnalytics();
-    // temp hack to fix the fact that the START button doesn't work
-    // this.goToBeginning();
```

Comment:
this should have never been committed; I'd added it a while ago while making other changes.